cluster/addons/fluentd-gcp/fluentd-gcp-image/run.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# For systems without journald
+mkdir -p /var/log/journal
+
+if [ -e /host/lib/libsystemd* ]
```

Comment:
Hi Mikhail,
This check results in an error, and as a result, the systemd libs are not copied at all. (I see that you completely remove this logic in https://github.com/kubernetes/kubernetes/pull/40017). But from my testing it seems that starting from this change the fluentd-gcp container does not run on GCI anymore.

Are you aware of this issue? What do you think about reverting https://github.com/kubernetes/kubernetes/pull/40017 and instead fixing this check so that this works again?

Thanks.